Test/Web/FunctionalTests/TestApps/AspNetDiagnostics/ApplicationInsights.config

```diff
@@ -42,10 +42,7 @@
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.UnobservedExceptionTelemetryModule, Microsoft.AI.WindowsServer"/>
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.FirstChanceExceptionStatisticsTelemetryModule, Microsoft.AI.WindowsServer"/>
   </TelemetryModules>
-  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel">
-    <EndpointAddress>http://localhost:4000/v2/track</EndpointAddress>
-    <MaxTelemetryBufferDelay>0:00:01</MaxTelemetryBufferDelay>
-  </TelemetryChannel>
+  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel"/>
```

Comment:
This will break the Func tests. (Ai.Config is destroyed after every nuget upgrade) Please revert this file to original.